03_letters/runnerdave/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	stats := make(map[rune]int)
+	for _, char := range s {
+		if freq, ok := stats[char]; ok {
+			stats[char] = freq + 1
+		} else {
+			stats[char] = 1
+		}
+	}
+	return stats
+}
+
+func sortLetters(m map[rune]int) []string {
```

Comment:
As far as i remember sorting should be done by `values` not by the `key`. You can find confirmation in the slack channel

Reply:
there is no specification on what to use to sort, from the **readme**:

> Write a function that returns a sorted slice of strings with elements "{key}:{val}". Use package sort:

The only spec is to use the package sort, I guess to practice using packages, which I did.

Reply:
👍